### PR TITLE
Added types to arraystat

### PR DESCRIPTION
**Pull request by dajuly20**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/arraystat/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Type definitions for arraystat 1.7
+// Project: https://www.npmjs.com/package/arraystat
+// Definitions by: Julian Wiche <https://github.com/dajuly20>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
```
Comment (peterblazejewicz) [inline types/arraystat/index.d.ts:9]:
```suggestion
```
remove those hints

File: types/arraystat/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    min: number;
+    max: number;
+    nb: number;
+}
+
+/**
+ * Executes a task as soon as possible.
+ * @param task Function or any object that implements `call()`.
+ */
+declare function arraystat(data: number[]): ArrayStatType;
```
Comment (peterblazejewicz) [inline types/arraystat/index.d.ts:35]: this module seems to ship only ES6 export for a single function, check here:
https://unpkg.com/browse/arraystat@1.7.75/index.js
The template used here was for UMD, please use ES6 module template:
https://www.typescriptlang.org/docs/handbook/modules.html#zipcodevalidatorts-1

Comment (jakebailey) [reply types/arraystat/index.d.ts:30]: Yeah, this should just be `export default function arraystat` and no `export =`.

**Comment by sandersn** [comment]
Looking at this again, it still needs to switch from `export = arraystat` to `export default arraystat`. (or `export default function arraystat(....`)

**Review by sandersn — CHANGES_REQUESTED** [review]
Requesting changes so the bot knows the state of the PR.

**Pull request closed without merging**